packages/core/app-layout/src/components/pageHeader/AppPageHeader.vue

```diff
@@ -86,6 +86,8 @@ const breadcrumbIconSlots = computed((): string[] => {
 
 <style lang="scss" scoped>
 .kong-ui-app-page-header {
+  margin-bottom: $kui-space-50;
```

Comment:
I believe this should be 20px of margin bottom (`$kui-space-70`) based on the chat with Josh/Jason yesterday 

@jillztom does that sound correct?

Reply:
I used same spacing that was used in `khcp-ui` before that but I thin 20px would look much better.

Reply:
Updated to use `$kui-space-70`

Reply:
If there is content in the bottom slot, does the bottom slot need default top margin? 

(Just commenting based on a convo yesterday; defer to Kai or Jilson for ✅ )

Reply:
No, below stol element has default margin top of `$kui-space-40`